src/Quiz.js

```diff
@@ -1,17 +1,110 @@
-import React from 'react';
-import Question from './Question.js';
-
-const Quiz = ({ quiz,  disabledQuest, match, onClick }) => {
-  const title = quiz.question;
-  const choices = quiz.incorrect_answers.concat(quiz.correct_answer);
-  const answer = quiz.correct_answer;
-  return <Question
-    match={match}
-    title={title}
-    choices={choices}
-    disabledQuest={disabledQuest}
-    onClick={(choice)=>onClick(choice, answer)}
-  />
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Card, Radio } from 'antd';
+import Score from './Score.js';
+import shuffle from 'shuffle-array';
+import InnerHTML from 'dangerously-set-inner-html';
+
+const RadioButton = Radio.Button;
+const RadioGroup = Radio.Group;
+
+class Quiz extends Component {
+  constructor(props){
+    const { quizId } = props.match.params;
+    super()
+    this.state={
+      questionsData: JSON.parse(window.localStorage.getItem(quizId)),
+      disabledQuestion:false,
+      streakBar: 0,
+      progressBar: [],
+    }
+  }
+
+  check = (choice, answer, quizId, questionId, lastQuestion) => {
```

Comment:
Since you have the questions in state and quizId/questionId in props you should only need to pass on the value. This is also where we can save to localStorage too.